xrdp/xrdp_wm.c

```diff
@@ -1279,21 +1281,23 @@ xrdp_wm_mouse_click(struct xrdp_wm *self, int x, int y, int but, int down)
                     self->mm->mod->mod_event(self->mm->mod, WM_BUTTON5UP,
                                              self->mouse_x, self->mouse_y, 0, 0);
                 }
-                if (but == 6 && down)
-                {
-                    self->mm->mod->mod_event(self->mm->mod, WM_BUTTON6DOWN, x, y, 0, 0);
-                }
-                else if (but == 6 && !down)
-                {
-                    self->mm->mod->mod_event(self->mm->mod, WM_BUTTON6UP, x, y, 0, 0);
-                }
-                if (but == 7 && down)
+
+                /* horizontal scroll */
+
+                if (but == 6)
```

Comment:
Can you add some constants to describe the values 6 , 7 ?

Reply:
Simply the values indicates mouse button. The constants will be MOUSE_BUTTON1 ... MOUSE_BUTTON7. Is it fine?

Reply:
I don't familiar with this, if they have a known position you can call them by its position .

Reply:
Defines here don't help much, 6 means button 6.  BUTTON6 not much better then 6.

Reply:
Right.